### PR TITLE
fix(ui): Fix large unread number not centered

### LINKED ISSUE #1376: improvement(badge): unread badge on chat isnt fully round
STR:
- sidebar
- get a message when not in chat
- unread badge appears
- is not fully rounded

Captura de ecrã 2023-10-23, às 16 28 47

macOS, m1

### PR DESCRIPTION
**Pull request by Flemmli97**
### What this PR does 📖

- Fixes large unread number (3 digit) not being centered

### Special notes for reviewers 🗒️

This PR #1376 should also be checked. I was not able to reproduce the problem in that PR but noticed this issue

### CODE REVIEW

**Comment by phillsatellite** [comment]
When tested it seemed like 2 digits had an oval shape to them while 3 digits was perfect 

Screenshot 2023-11-03 at 12 45 07 PM
Screenshot 2023-11-03 at 12 44 34 PM

**Comment by Flemmli97** [comment]
hmm. this is very strange. so its for you too?
but for me this never happens

Screenshot 2023-11-03 at 18 35 38


what os you on?

and can you show the dimensions of the badge? like inspect and hover over it for the pixels

File: ui/src/style.scss [hunk 1]
```diff
@@ -218,6 +218,12 @@ body {
   right: -4px; 
   padding: 2px;
   text-align: center;
+  display: flex;
+  span {
```
Comment (lgmarchi) [inline ui/src/style.scss]: We have other span in main.rs, I think it is better to create a class for this case.

**Review by dariusc93 — APPROVED** [review]

**Comment by phillsatellite** [comment]
Screenshot 2023-11-07 at 6 37 56 PM

**Comment by phillsatellite** [comment]
Any update on this PR?

**Comment by stavares843** [comment]
cc @Flemmli97

File: ui/src/lib.rs [hunk 2]
```diff
@@ -106,6 +106,11 @@ pub static WINDOW_CMD_CH: Lazy<WindowManagerCmdChannels> = Lazy::new(|| {
     }
 });
 
+const UNREAD_ASPECT_RATIO: &str = r#"for (element of document.getElementsByClassName("nav-unread-indicator")) {
+    console.log("EL ", element, " h ", element.getBoundingClientRect().width)
```
Comment (stavares843) [inline ui/src/lib.rs]: do we need the console log?

Comment (Flemmli97) [reply ui/src/lib.rs]: removed

**Pull request merged**